File: Modul-4/C/PRAK404-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int pilihan;
  float nilai1, nilai2;

  while (1)
  {
    printf("Pilih program\n");
    printf("1. Penjumlahan\n");
    printf("2. Pengurangan\n");
    printf("3. Perkalian\n");
    printf("4. Pembagian\n");
    printf("5. Exit\n");

    printf("Masukkan Pilihan : ");
    scanf("%d", &pilihan);

    if (pilihan >= 1 && pilihan <= 4)
    {
      printf("Masukkan nilai pertama : ");
      scanf("%f", &nilai1);

      printf("Masukkan nilai kedua : ");
      scanf("%f", &nilai2);

      switch (pilihan)
      {
      case 1:
        printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1 + nilai2);
        break;
      case 2:
        printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1 - nilai2);
        break;
      case 3:
        printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1 * nilai2);
        break;
      case 4:
        if (nilai2 != 0)
        {
          printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1 / nilai2);
        }
        else
        {
          printf("Error: Tidak bisa membagi oleh 0\n");
        }
        break;
      }
    }
    else if (pilihan == 5)
    {
      printf("Terimakasih, telah menggunakan kalkulator Bintang\n");
      break;
    }
    else
    {
      printf("Input anda salah, silahkan coba lagi\n");
    }
  }

  return 0;
}

File: Modul-4/Python/PRAK404-2310817110006-Adrian_Bintang_Saputera.py
while True:
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")

    pilihan = int(input("Masukkan Pilihan : "))

    if 1 <= pilihan <= 4:
        nilai1 = float(input("Masukkan nilai pertama : "))
        nilai2 = float(input("Masukkan nilai kedua : "))

        if pilihan == 1:
            print(f"Hasil Penjumlahan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1 + nilai2:.2f}")
        elif pilihan == 2:
            print(f"Hasil Pengurangan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1 - nilai2:.2f}")
        elif pilihan == 3:
            print(f"Hasil Perkalian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1 * nilai2:.2f}")
        elif pilihan == 4:
            if nilai2 != 0:
                print(f"Hasil Pembagian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1 / nilai2:.2f}")
            else:
                print("Error: Tidak bisa membagi oleh 0")
    elif pilihan == 5:
        print("Terimakasih, telah menggunakan kalkulator Bintang")
        break
    else:
        print("Input anda salah, silahkan coba lagi")